### ISSUE TITLE
W504 on PEP570 slash

### ISSUE THREAD

**Issue by rharish101** [PyCQA/pycodestyle#951]
Configuraton:
* OS: Arch Linux
* pycodestyle version: 2.6.0
* Python version: 3.8.3
* Installation source: Arch Linux packages

W504 (line break after binary operator) shows up when the slash that indicates positional-only arguments ([PEP 570](https://www.python.org/dev/peps/pep-0570/)) is followed by a newline. For example:
```python
def really_long_name_for_a_function(
    very_long_argument_1, very_long_argument_2, /
):
    pass
```

pycodestyle thinks that the slash is a binary operator. I can, however, silence it using a comma after the slash, as follows:
```python
def really_long_name_for_a_function(
    very_long_argument_1, very_long_argument_2, /,
):
    pass
```

I think this is an issue, as the slash isn't a binary operator, and the comma seems like a workaround for arguments on a single line than good practice (at least in this case).

**Comment by asottile** [comment]
this also triggers with keyword-only arguments:

```console
$ cat t.py 
def really_long_name_for_a_function(
    very_long_argument_1, very_long_argument_2, *
):
    pass
$ ~/opt/venv/bin/pycodestyle t.py --select=W503,W504
t.py:2:49: W504 line break after binary operator
```

**Comment by DevilXD** [comment]
Just ran into this myself. Simple snippet for reproduction:

```py
def func1(arg1, arg2, arg3, /):  # no error
    pass


def func2(
    arg1, arg2, arg3, /  # W504 line break after binary operator
):
    pass
```

**Comment by sigmavirus24** [comment]
Thanks @DevilXD I think we already had enough examples for reproduction though and don't need further ones.
